scripts/verify_commits.sh

```diff
@@ -37,9 +37,12 @@ function verify_downstream_only() {
     local inside_staging
     inside_staging="$(git show --name-only "${downstream_commit}" -- staging)"
     if [[ -n "${inside_staging}" ]]; then
-        err "downstream non-staging commit ${downstream_commit} changes staging"
+        if git log -n 1 "${downstream_commit}" --pretty=%s | grep -q '[CARRY]'; then
```

Comment:
Why `[CARRY]`?

Reply:
I don't have a strong opinion on this. Carry is the word OCP uses for commits that are carried on top of upstream for the forseeable future.

Reply:
I'm ambivalent, but this ought to be documented somewhere.

Reply:
Where would you suggest? As it stands the failure messages from the script are fairly explicit - if you hit this case by writing a commit that you mean to carry, you will see error messages telling you how to change the commit message. Happy to add it to some other location you think folks will be likely to read.

Reply:
I have no suggestions for an alternative; but mostly just document that commits with "whatever-prefix-is-finally-chosen" in the headline are treated special. Either in the `README.md` in the `scripts` directory or in Confluence.